src/streams/StreamItem.js

```diff
@@ -151,13 +161,24 @@ export default function StreamItem(props: Props): Node {
       }}
     >
       <View style={wrapperStyle}>
+        {isCollapsible === true && (
+          <Touchable onPress={handleCollapse && (() => handleCollapse(streamId))}>
+            <CollapseIcon
+              style={componentStyles.collapseIcon}
+              size={20}
+              color={iconColor}
+              isOpen={isUnCollapsed}
+            />
```

Comment:
```suggestion
            {isUnCollapsed === true ? (
              <IconUnCollapse color={color} size={20} style={componentStyles.collapseIcon} />
            ) : (
              <IconCollapse color={color} size={20} style={componentStyles.collapseIcon} />
            )}
```

I think we don't need this new `CollapseIcon` component to render the icon. The icon and the `Touchable` work closely together to give the user a collapse/uncollapse button. With the icon defined in a separate file, we're asking the reader to jump through the `CollapseIcon` reference just to see how this button is implemented.